estate/models/estate_property.py

```diff
@@ -0,0 +1,100 @@
+from dateutil import relativedelta
+
+from odoo import _, api, exceptions, fields, models
```

Comment:
We usually import directly, but ok as is
```suggestion
from odoo import _, api, fields, models
from odoo.exceptions import UserError, ValidationError
```